### PR TITLE
Feature/20 swagger UI

### PR DESCRIPTION
**Pull request by PaulWen**
image

### CODE REVIEW

**Comment by PaulWen** [comment]
@holzeis Can we just add the full `.idea` folder to the `.gitignore`? Currently, only selected files from `.idea` are excluded.

**Comment by PaulWen** [comment]
@holzeis Should we exclude `generator.iml` from the repo?

**Comment by holzeis** [comment]
@PaulWen well I've followed the default gitignore settings from gitignore.io. Why do you propose excluding the full .idea folder as well as generator.iml. Particularily for the idea i find it useful as it will also include the license header settings.

File: generator.iml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+    </facet>
     <facet type="kotlin-language" name="Kotlin">
-      <configuration version="4" platform="JVM 17" allPlatforms="JVM [17]" useProjectSettings="false">
+      <configuration version="4" platform="JVM 11" allPlatforms="JVM [11]" useProjectSettings="false">
         <compilerSettings>
           <option name="additionalArguments" value="-java-parameters -Xjsr305=strict" />
         </compilerSettings>
         <compilerArguments>
-          <option name="jvmTarget" value="17" />
+          <option name="jvmTarget" value="11" />
```
Comment (holzeis) [inline generator.iml:22]: was this downgrade intentional? if yes why?

File: pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	<name>generator</name>
 	<description>Load Generator for Aca-Py</description>
 	<properties>
-		<java.version>17</java.version>
+		<java.version>11</java.version>
```
Comment (holzeis) [inline pom.xml:17]: was this downgrade intentional? if yes why?

Comment (PaulWen) [reply pom.xml:17]: Yes, Coba currently supports Java 11 only.
Do you currently see any downside to using Java 11?

**Pull request merged**